base_user_role/tests/test_user_role.py

```diff
@@ -105,3 +105,61 @@ def test_role_1_2_with_dates(self):
         role1_group_ids.append(role1.group_id.id)
         role_group_ids = sorted(set(role1_group_ids))
         self.assertEqual(user_group_ids, role_group_ids)
+
+    def test_role_unlink(self):
+        # Get role1 groups
+        role1 = self.role_model.browse(self.cr, self.uid, self.role1_id)
+        role1_group_ids = role1.implied_ids.ids
+        role1_group_ids.append(role1.group_id.id)
+        role1_group_ids = sorted(set(role1_group_ids))
+        # Get role2 groups
+        role2 = self.role_model.browse(self.cr, self.uid, self.role2_id)
+        role2_group_ids = role2.implied_ids.ids
```

Comment:
I see that `role2_group_ids` was not used anywhere below, can we remove it?

Reply:
@novawish indeed, fixed